contrib/scalar/scalar.c

```diff
@@ -116,6 +116,14 @@ static int set_recommended_config(void)
 		{ "core.multiPackIndex", "true" },
 		{ "core.preloadIndex", "true" },
 		{ "core.safeCRLF", "false" },
+#ifdef HAVE_FSMONITOR_DAEMON_BACKEND
+		/*
+		 * The value proposition of Scalar is really only realized by
+		 * using FSMonitor. We choose to only support the built-in
+		 * FSMonitor, and only enable this on supported platforms.
```

Comment:
I'm not sold on this comment about "the value proposition". This is _one part_ of that. It is one of the many opinions that Scalar auto-configures when available.

Reply:
Should I drop just the "value proposition", leaving the `We choose to only support the built-in FSMonitor, and only enable this on supported platforms.` part.. or drop the entire comment? @derrickstolee

Reply:
I think a comment like "Enable the builtin FS Monitor on supported platforms." would be good.